InstallRHELandClones.md

```diff
@@ -79,7 +55,13 @@ sudo yum install epel-release
 sudo yum install freetds-devel
 ```
 
-## IMPORTANT: CentOS7/Oracle7 and PostgreSQL >= 11
+Some other dependencies are also needed to install PostgreSQL and then compile tds_fdw:
+
+```bash
+sudo yum install clang llvm make redhat-rpm-config wget
+```
+
+#### IMPORTANT: CentOS7/Oracle7 and PostgreSQL >= 11
```

Comment:
Let's not remove CentOS7 for now. It's EoL, but there are still some alive clones there, and the PostgreSQL development group still list it at their installation instructions.